test/unit/test_snowflake_adapter.py

```diff
@@ -523,3 +524,42 @@ def test_float_from_description(self):
         assert col.is_numeric() is False
         assert col.is_string() is False
         assert col.is_integer() is False
+
+
+class SnowflakeConnectionsTest(unittest.TestCase):
+    def test_comment_stripping_regex(self):
+        pattern = r'(\".*?\"|\'.*?\')|(/\*.*?\*/|--[^\r\n]*$)'
+        comment1 = '-- just comment'
+        comment2 = '/* just comment */'
+        query1 = 'select 1; -- comment'
+        query2 = 'select 1; /* comment */'
+        query3 = 'select 1; -- comment\nselect 2; /* comment */ '
+        query4 = 'select \n1; -- comment\nselect \n2; /* comment */ '
```

Comment:
Could you add one more unit test that represents the original error case reported in #2731:
```
query5 = 'select 1; -- comment \nselect 2; -- comment \nselect 3; -- comment'
expected_query_5 = 'select 1; \nselect 2; \nselect 3;'
```

Reply:
I've added the extra test. Let me know if there's anything more